cli/compose/interpolation/interpolation.go

```diff
@@ -42,7 +42,7 @@ func interpolateSectionItem(
 		if err != nil {
 			return nil, errors.Errorf(
 				"Invalid interpolation format for %#v option in %s %#v: %#v. You may need to escape any $ with another $.",
-				key, section, name, err.Template,
+				key, section, name, err.(*template.InvalidTemplateError).Template,
```

Comment:
We might want to be a little bit safer here than just this type assertion, but at that point, it can't be something else anyway.